src/notation/view/widgets/editstyle.ui

```diff
@@ -235,7 +240,7 @@
      </widget>
      <widget class="QStackedWidget" name="pageStack">
       <property name="currentIndex">
-       <number>0</number>
+       <number>35</number>
```

Comment:
This change is not needed. Probably harmlsee too though, IIRC it gets set to 1 programatically anyway

Reply:
Resolved